### ISSUE TITLE
Caseflow > Switch product should be an expanding/collapsing button and not a link

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12124]
The Switch Product element is incorrectly read as a link for assistive technology users. Functionally this is an expanding and collapsing element and should be read as an expanding and collapsing   button. (New: 08/20/2019)

**Comment by leikkisa** [comment]
@leikkisa look for screenshot

**Comment by alisan16** [comment]
This task refers to the Switch Product button in the header and what html element it's being implemented with.

<img width="1339" alt="Screen Shot 2019-11-18 at 1 57 13 PM" src="https://user-images.githubusercontent.com/1198106/69081147-61cd8b80-0a0b-11ea-9198-8223ecfa49c5.png">
